src/components/Papers/PapersList.jsx

```diff
@@ -103,28 +123,88 @@ const PapersList = ({ history, match }) => {
         </CozyTheme>
       </BarCenter>
 
-      <List>
-        {paperslistByContact?.map((paperByContact, idx) => (
-          <Fragment key={idx}>
-            <ListSubheader classes={isMobile && classes}>
-              {paperByContact.contact}
-            </ListSubheader>
-            <div className={'u-pv-half'}>
-              {paperByContact.papers.map((paper, idx) => (
-                <PaperLine
-                  key={idx}
-                  paper={paper}
-                  divider={idx !== paperByContact.papers.length - 1}
-                  actions={actions}
-                />
-              ))}
-            </div>
-          </Fragment>
-        ))}
-      </List>
+      {paperslistByContact.length > 0 ? (
+        <List>
+          {paperslistByContact?.map(({ contact, papers }, idx) => (
+            <Fragment key={idx}>
+              <ListSubheader classes={isMobile && classes}>
+                {contact}
+              </ListSubheader>
+              <PapersListByContact papers={papers} />
```

Comment:
This naming is a bit confusing. To me `<PapersListByContact` should take `paperslistByContact` from props. 

Here this is more a `PapersList` ?